content/en/observability_pipelines/destinations/_index.md

```diff
@@ -43,6 +79,14 @@ Set up the Amazon OpenSearch destination and its environment variables when you
 
 {{% observability_pipelines/configure_existing_pipelines/destination_env_vars/amazon_opensearch %}}
 
+### Batch settings
```

Comment:
For all of the below sections where the batch settings is being mentioned on a per component basis: I think we should refrain from mentioning them there. 🤔  Perhaps we should mention only in one place and not both ?

Reply:
@neuronull after the Logs-to-metrics template comes out (and I'm assuming there is going to be a Datadog metrics destination for it that is similar to the Datadog Logs destination), I'm going to make each destination it's own page in the docs (like the [Amazon S3 archive](https://docs.datadoghq.com/observability_pipelines/destinations/amazon_s3/?tab=docker) destination), so that's why I put them here. But I'm fine with removing it for this first iteration, because it does seem quite redundant.

Just wondering if you have thoughts about adding the batch settings for each destination once they have their own doc?

Reply:
I just added the batch settings for the Amazon S3 destination doc as an example.
Preview: https://docs.datadoghq.com/observability_pipelines/destinations/amazon_s3/?tab=docker#batch-settings-intro

Commit: https://github.com/DataDog/documentation/pull/24821/commits/8a52d29c729e2d674dae31c913a381be6193cade

Reply:
> and I'm assuming there is going to be a Datadog metrics destination for it that is similar to the Datadog Logs destination

correct, there will be one

Reply:
> Just wondering if you have thoughts about adding the batch settings for each destination once they have their own doc?

If they each have their own page, yeah makes sense to me. Then each page has its own batch parameters, and a link back to the main description of the batching behavior. Makes sense. (and the main page describing batching behavior wouldn't have the table that contains everything).

So I guess, maybe that could be a solution for this PR? add the individual parameters in each section but not have the large table. WDYT ?

Reply:
> So I guess, maybe that could be a solution for this PR? add the individual parameters in each section but not have the large table. WDYT ?

@neuronull Yes, I agree, that seems like the better solution here. I kept the individual parameters, and removed the large table.

> I used that term in the internal confluence page because in the sink configuration, they are truly settings. But in OP we do not expose them as something the user can change. So I'm wondering if we should ajust the terminology and just call it "Batching behavior". , and remove/adjust any language that suggests it could be configurable.

I forgot to respond to this earlier. That's a good point and makes sense. I changed the header from `Batch settings` to `Event batching`, and used `batching behavior` or `batching parameters` in the explanation. Does that work?

Commit: https://github.com/DataDog/documentation/pull/24821/commits/64d9e4b2ae6ae6fdb89122d711a13ca678447c8b

And a little rewording: https://github.com/DataDog/documentation/pull/24821/commits/c9351402af51ba8630b2df72f9a21153f9de52da

> I'm curious if Product/Design might have input on the usage of the term "settings" in these changes.

I'll check in with Pratik about all of this when he's back.

Reply:
Yeah, looks great!

Reply:
Thanks for the review and feedback!

Reply:
yw!